recipe/meta.yaml

```
@@ -57,7 +57,7 @@ requirements:
     - shap ==0.42.1
     - mlflow >=2.1.0,<2.4
     - sentencepiece >=0.1.95,<0.2
```

```suggestion
    - peft >=0.5.0,<1
    - sentencepiece >=0.1.95,<0.2
```
You can add it to run_constrained even if it's not available yet

---

yes as we have biweekly builds, lets do this next time